sdk/typespec/typespec_client_core/Cargo.toml

```diff
@@ -58,6 +59,7 @@ reqwest_rustls = [
 test = [] # Enables extra tracing including error bodies that may contain PII.
 tokio = ["tokio/fs", "tokio/sync", "tokio/time", "tokio/io-util"]
 xml = ["dep:quick-xml"]
+rust_decimal = ["dep:rust_decimal"]
```

Comment:
Nit: You probably don't need to use the crate name as the feature name - you can just use `decimal` instead (similar to how we add `http` and `json` to express constructs.

Reply:
Thank you! Done.